posts/inside-rust/2020-10-14-regression-labels.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Marking issues as regressions"
+description: "Now anyone can mark issues as regressions!"
+author: Camelid
+team: the release team <https://www.rust-lang.org/governance/teams/operations#release>
+---
+
+If you've contributed to Rust before, you have likely used a command like this
+to [set labels][rustbot-labeling-docs]:
```

Comment:
I think we should avoid starting off like this. People who haven't used it might feel excluded. Plus, I think this post should actually be moved to the main blog: inside rust is targeted to contributors, but we want regular users to do this as well.

Reply:
> I think we should avoid starting off like this. People who haven't used it might feel excluded.

That's a good point, I didn't think of that. Perhaps I'll go with something like "One of the ways we keep track of Rust issues is with labels, which you can modify using a command like this...". Or maybe it would make more sense to just skip this part?

> Plus, I think this post should actually be moved to the main blog: inside rust is targeted to contributors, but we want regular users to do this as well.

Sounds good, I was thinking of suggesting that.